embeddings/evaluator/submission.py

```diff
@@ -28,6 +28,58 @@ class Submission:
     packages: List[str]
     config: Optional[Dict[str, Any]] = None  # any additional config
 
+    @staticmethod
+    def _init_evaluator_kwargs(hparams: Dict[str, Any]) -> Dict[str, Any]:
+        evaluator_kwargs = {}
+        evaluation_mode = hparams.get("evaluation_mode", None)
+        tagging_scheme = hparams.get("tagging_scheme", None)
+
+        if evaluation_mode or tagging_scheme:
+            evaluator_kwargs.update(
+                {
+                    "evaluation_mode": evaluation_mode,
+                    "tagging_scheme": tagging_scheme,
+                }
+            )
+        return evaluator_kwargs
+
+    @staticmethod
+    def from_local_disk(
```

Comment:
can use `classmethod` instead since we use methods defined in the `Submission` class

Reply:
done